src/api/app/models/event/base.rb

```diff
@@ -310,19 +310,17 @@ def shorten_payload_if_necessary
 #
```

Comment:
Same here. The commit message should state why a change was made.

Reply:
what do you mean? 🤔

Reply:
The commit message says "Remove `events.project_logged column` as it is not needed any more.", but that does not tell why:-)
For commit messages the why is usually the interesting part. What has been changed can be grasped from the commit diff, but not the why.

Let's say you have to debug some bug and stumble over some commit that might have introduced the bug. You can get what the commit did easily from the diff, but you won't know the reason behind these changes, unless it was documented in the commit message.
This often seems not so important, but every once in a while it might save you hours of debugging:-)